application/cms/views.py

```diff
@@ -1109,3 +1112,167 @@ def view_measure_version_by_measure_version_id(measure_version_id):
             version=measure_version.version,
         )
     )
+
+
+@cms_blueprint.route(  # noqa: C901 (complexity)
```

Comment:
We shouldn't need the `noqa` on these lines - remove them wherever possible please :)

Reply:
👍  (bad copy and pasting on my part).